monkey/infection_monkey/island_api_client/http_client.py

```diff
@@ -60,22 +60,13 @@ def __init__(self, retries=RETRIES):
         retry_config = Retry(retries)
         self._session.mount("https://", HTTPAdapter(max_retries=retry_config))
         self._api_url: Optional[str] = None
+        self._headers = {}
```

Comment:
Probably should be `None`

Reply:
Any reason? Empty dict makes sense to me.